### PR TITLE
Implement raw to curated for Episode

### PR DESCRIPTION
**Pull request by schammass**

### CODE REVIEW

File: src/main/scala/bio/ferlab/ui/etl/coda19/TerminologyETL.scala [hunk 1]
```diff
@@ -26,6 +26,8 @@ class TerminologyETL(salt: String)(implicit conf: Configuration) extends ETL {
         col("pcrName.loinc_code").as("display_code"),
         col("pcrName.loinc_reference_url").as("reference_url"))
 
+    pcrNameDF.show(false)
```
Comment (botekchristophe) [inline src/main/scala/bio/ferlab/ui/etl/coda19/TerminologyETL.scala:29]: remove show()

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      .drop("lifetimenumber")
+
+
+    ptb.createOrReplaceTempView("ptb")
+
+    val ptb_with_hash = spark.sql(
+      s"""
+         |SELECT pbkdf2Udf(bedid, '$salt') bedid, bedlabel, utcouttime, utcintime, encounterid FROM ptb
+         |""".stripMargin)
+      .withColumn("display", col("bedlabel")) //TODO remove ptb.
```
Comment (botekchristophe) [inline src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala:50]: remove comment here

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      .withColumn("place", lit("Location"))
+      .withColumn("reference", concat_ws("/", col("place"), col("bedid")))
+      .withColumn("location", struct("display", "reference"))
+      .withColumn("end", col("ptb.utcouttime"))
```
Comment (botekchristophe) [inline src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala:54]: remove all 'pbt.'

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      .withColumn("start", col("ptb.utcintime"))
+      .withColumn("period", struct("end", "start"))
+      .withColumn("status",
+        when(col("period.start").isNull and col("period.end").isNull, lit("planned"))
+          .when(col("period.start").isNotNull and col("period.end").isNull, lit("in progress"))
```
Comment (botekchristophe) [inline src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala]: in-progress

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+
+    ptb.createOrReplaceTempView("ptb")
+
+    val ptb_with_hash = spark.sql(
+      s"""
+         |SELECT pbkdf2Udf(bedid, '$salt') bedid, bedlabel, utcouttime, utcintime, encounterid FROM ptb
+         |""".stripMargin)
+      .withColumn("display", col("bedlabel")) //TODO remove ptb.
+      .withColumn("place", lit("Location"))
```
Comment (botekchristophe) [inline src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala:51]: change with 
.withColumn("subject", struct(concat_ws("/", lit("Patient"), col("lifetimenumber")) as "reference"))

File: src/main/scala/bio/ferlab/ui/etl/coda19/TerminologyETL.scala (hunk 1 above)
Comment (schammass) [reply src/main/scala/bio/ferlab/ui/etl/coda19/TerminologyETL.scala:29]: ok. tks

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala (hunk 2 above)
Comment (schammass) [reply src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala]: done

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala (hunk 3 above)
Comment (schammass) [reply src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala]: done

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala (hunk 4 above)
Comment (schammass) [reply src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala]: done

File: src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala (hunk 5 above)
Comment (schammass) [reply src/main/scala/bio/ferlab/ui/etl/coda19/EncounterETL.scala]: done

**Comment by schammass** [comment]
I updated all requests and pushed -f

**Review by botekchristophe — APPROVED** [review]

**Pull request merged**